.azure-pipelines/benchmarks.yml

```diff
@@ -80,7 +80,7 @@ stages:
       inputs:
         command: 'run'
         projects: '$(System.DefaultWorkingDirectory)/test/benchmarks/Benchmarks.Trace/Benchmarks.Trace.csproj'
-        arguments: '-c Release -f netcoreapp3.1 -- -r net472 netcoreapp3.1 -m -f * --iterationTime 1500'
+        arguments: '-c Release -f netcoreapp3.1 -- -r net472 netcoreapp3.1 -m -f * --iterationTime 2000'
```

Comment:
This needs to be updated in the ultimate-pipeline too

Reply:
Looks like I forgot to decrease the iteration time in the ultimate-pipeline, so I don't need to revert it 😄